app/bundles/CoreBundle/Doctrine/DatabasePlatform.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Doctrine;
+
+use Doctrine\DBAL\Platforms\AbstractMySQLPlatform;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\DB2Platform;
+use Doctrine\DBAL\Platforms\OraclePlatform;
+use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
+use Doctrine\DBAL\Platforms\SqlitePlatform;
+use Doctrine\DBAL\Platforms\SQLServerPlatform;
+
+class DatabasePlatform
+{
+    public static function getDatabasePlatform(AbstractPlatform $platform): string
+    {
+        // @todo remove in mautic 6.
```

Comment:
What should be removed in M6? Just the next if statement or the whole method? Mysql is the only supported database platform. Why do we need others except mysql?

Reply:
Because the `\Doctrine\DBAL\Platforms\AbstractPlatform::getName` is deprecated, and i'm pretty sure this class should be removed in 6.0, and we should implement methods related to the platform before the M6. Or otherwise leave as it is, and have a tech debt.

Reply:
So this comment belongs to the whole class? Can it be moved up to line 14 with some more details?

Reply:
Looked back to the code, and it seems we require the string representation of the database platform used, so i removed a todo.